tests/unit/amo/sagas/test_users.js

```diff
@@ -276,6 +338,10 @@ describe(__filename, () => {
   describe('fetchUserNotifications', () => {
     it('calls the API to fetch the notifications of a user', async () => {
       const username = 'tofumatt';
+
+      const user = createUserAccountResponse({ username });
+      sagaTester.dispatch(loadCurrentUserAccount({ user }));
```

Comment:
I have added this because I saw an `invariant` error in the logs (but it did not fail the test case because the test case).